backend/routers/spiritual.py

```
@@ -336,68 +336,106 @@ async def get_birth_chart(request: Request):
+        logger.info(f"Language: {language}")
+
+        if not user_question or not birth_details:
+            logger.error("❌ Missing question or birth details")
+            raise HTTPException(status_code=400, detail="Missing question or birth details")
+
+        date = birth_details.get("date")
+        time_ = birth_details.get("time")
+        location = birth_details.get("location")
+        latitude = "9.66845"
+        longitude = "80.00742"
+        timezone = "Asia/Colombo"
```

Ruff (0.11.9)

Local variable `timezone` is assigned to but never used

Remove assignment to unused variable `timezone`

(F841)